Discover/DiscoverRouter.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DiscoverRouter.swift
+//  Discover
+//
+//  Created by Егор Бадмаев on 27.10.2022.
+//  
+//
+
+import UIKit
+import Models
+
+final class DiscoverRouter {
+    weak var output: DiscoverRouterOutput?
+    weak var viewController: UIViewController?
+}
+
+extension DiscoverRouter: DiscoverRouterInput {
+    
+    func openRecipeDetailsModule(for recipe: Recipe) {
+        
```

Comment:
Create _RecipeDetails_ module and implement routing to it. So we won't need to create one more PR for this change